static/app/views/replays/detail/userActionsNavigator.tsx

```diff
@@ -108,16 +94,14 @@ function UserActionsNavigator({event, crumbs}: Props) {
               key={item.id}
               onMouseEnter={() => onMouseEnter(item)}
               onMouseLeave={() => onMouseLeave()}
-              onClick={() => {
-                setCurrentUserAction(item);
-                item.timestamp
-                  ? setCurrentTime(relativeTimeInMs(item.timestamp, startTimestamp))
-                  : '';
-              }}
             >
               <Container
                 isHovered={closestUserAction?.id === item.id}
                 isSelected={currentUserAction?.id === item.id}
+                onClick={() => {
+                  item.timestamp &&
+                    setCurrentTime(relativeTimeInMs(item.timestamp, startTimestamp));
+                }}
```

Comment:
This must go back to using a ternary `a ? b : c` instead of the AND-operator `a && b`. 

```suggestion
                onClick={() => 
                  item.timestamp
                    ? setCurrentTime(relativeTimeInMs(item.timestamp, startTimestamp))
                    : null;
                }
```

**Why?**

The AND operator is troublesome when the condition expression `a` returns a number. In this case our expression is `item.timestamp` will be a number. So we have to watch out.

Numbers break things because `0 === false`, because javascript. 🦶 🔫 .
- The AND operator will evaluate the expression to be `0`.
- React will insert `0` to the DOM.
- We will see `0` floating randomly in the html output.

It's actually happening already in another line in the code and looks like this:

<img width="361" alt="Screen Shot 2022-05-18 at 10 51 23 AM" src="https://user-images.githubusercontent.com/187460/168947974-7f9d91b1-c53f-4aea-acc5-91b9253692a0.png">